.travis.yml

```diff
@@ -1,18 +1,18 @@
 ---
-sudo: false
 language: ruby
+sudo: false
 cache: bundler
 rvm:
-- 2.4.4
-- 2.3.6
+- 2.4.5
 addons:
   postgresql: '9.5'
-env:
-  global:
-  - RAILS_ENV=test
-  - DATABASE_URL="postgresql://root:smartvm@localhost:5432/topological_inventory_test?encoding=utf8&pool=5&wait_timeout=5"
 before_install:
-- gem install bundler -v 1.16.4
-- psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
+- gem install bundler
+- DATABASE_URL="postgresql://postgres:@localhost:5432/topological_inventory_test?encoding=utf8&pool=5&wait_timeout=5"
```

Comment:
@agrare I think this repo does something where DATABASE_URL is not honored in specs...we are trying to standardize on this for test purposes for the various projects, so we will need to figure this out...I can remove it temporarily if needed.